### DISCUSSION TITLE
Join OpenSubsonic?

### DISCUSSION THREAD

**Discussion by deluan** [tamland/airsonic-refix#96] [Ideas]
Hey @tamland, this is @deluan, Navidrome's developer. Sorry for opening a discussion for this, but I couldn't find a way to contact you directly.

Not sure if you know, but some Subsonic server and client devs are joining forces to form an effort to evolve the Subsonic API: https://github.com/opensubsonic/open-subsonic-api/discussions Are you interested in joining as a member?

**Comment by tamland** [comment]
Sure thing. Sounds like a great initiative, happy to join.
